API.md

```diff
@@ -316,17 +316,18 @@ A low-level interface to geocoding, useful for more complex uses and reverse-geo
 | Options | Value | Description |
 | ---- | ---- | ---- |
 | id _or_ url | string | Value must be <ul><li>A [geocoder index ID](https://www.mapbox.com/developers/api/geocoding/), e.g. `mapbox.places`</li><li>A geocoder API URL, like `{{site.tileApi}}/v4/geocode/mapbox.places/{query}.json`</li></ul> |
-| options | Object | The second argument is optional. If provided, it may include: <ul><li>`accessToken`: Mapbox API access token. Overrides `L.mapbox.accessToken` for this geocoder.</li></ul> |
+| options | Object | The second argument is optional. If provided, it may include: <ul><li>`accessToken`: Mapbox API access token. Overrides `L.mapbox.accessToken` for this geocoder.</li><li>`proximity`: boolean which enables viewport based proximity search</li></ul> |
 
 _Returns_ a `L.mapbox.geocoder` object.
 
-### geocoder.query(queryString, callback)
+### geocoder.query(queryString|options, callback)
```

Comment:
@tmcw This is what I had originally but @jfirebaugh pointed out this is part of the public API and would therefore be a breaking change. In `3.0` I'd be in favour of switching to this.